### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by zancas**
These tests make this project more hackable, by fixing and eliminating noisy errors.

* tests all pass
* the parser is modified
* the obscuring error is fixed
* the `npx browserslist@latest --update-db` has been run.

This PR includes changes to app code!

### CODE REVIEW

**Comment by zancas** [comment]
I need to go over the legacy query-params processing carefully, to make sure I didn't short-circuit valid behavior.

In particular I notice the word `Duplicate`, that needs to be checked into.

**Review by AloeareV — COMMENTED** [review]
I'm not approving this because it's marked as draft, but this all looks pretty good!

File: src/utils/parse_zaddrs.test.js [hunk 1]
```diff
@@ -65,53 +65,61 @@ describe("exercise parseZcashURI", () => {
...
   });
 
-  test("invalid parameter queryarg name", () => {
+  test("invalid index", () => {
     // bad index
     const error = parseZcashURI(
       "zcash:tmEZhbWHTpdKMw5it8YDspUXSMGQyFwovpU?amount=2&address.a=tmEZhbWHTpdKMw5it8YDspUXSMGQyFwovpU&amount.a=3"
     );
     expect(typeof error).toBe("string");
+    expect(error).toBe("Duplicate param address");
```
Comment (AloeareV) [inline src/utils/parse_zaddrs.test.js:113]: Why are we expecting "Duplicate param address" in test "invalid index"?

Comment (zancas) [reply src/utils/parse_zaddrs.test.js:113]: I think that's stale.

Comment (AloeareV) [reply src/utils/parse_zaddrs.test.js:113]: It's the latest commit pushed, at least.

Comment (zancas) [reply src/utils/parse_zaddrs.test.js:113]: We have open questions about this, that I am going to issue and focus on.  In the interim, I vote we land this on `dev` and work against it.

File: src/App.tsx [hunk 2]
```diff
@@ -1,20 +1,13 @@
 /* eslint-disable import/no-webpack-loader-syntax */
-import React from "react";
```
[the remark this answers is not in the document]
Comment (zancas) [reply src/App.tsx:2]: Looks like not?!

**Comment by zancas** [comment]
I am tracking the remaining work that needs to be completed on the allowed query args here:

https://github.com/zingolabs/zecwallet-lite/issues/8

**Comment by dannasessha** [comment]
I can confirm all tests are passing for me locally, unlike the `dev` branch.

**Comment by zancas** [comment]
Fixes: #5

**Comment by zancas** [comment]
After reviewing this https://zips.z.cash/zip-0321#uri-syntax I am closing this PR in favor of a more incremental approach.

**Pull request closed without merging**